jenkins/pull_request_creator.py

```diff
@@ -48,15 +48,14 @@ def _set_github_data(self):
         self.github_instance = self._get_github_instance()
         self.user = self._get_user()
 
-        # Last folder in repo_root should be the repository
-        directory_list = self.repo_root.split("/")
-        self.repository_name = directory_list[-1]
-        LOGGER.info("Trying to connect to repo: {}".format(self.repository_name))
+        LOGGER.info("Trying to connect to repo")
```

Comment:
Does it make sense to include `repo_root`, or anything else here?  I'm not actually sure what is in that and whether it would be useful.

Reply:
I'm not actually sure the logging statement is useful anymore. The new lookup process doesn't leave a very natural gap in between "determine a plausible name" and "try to access the repo", which is why I ended up adding a logging statement to `repo_from_remote` instead. I considered splitting that function into the two parts, but frankly I want to avoid as much as possible anything that affects the stack of `@patch` mocking decorators in the tests...